doc/ref_model/chapters/chapter08-annex.md

```diff
@@ -6,232 +6,133 @@
-</ul>
-</li>
-<li>Validate ceph
-<ul>
-<li>Restart ceph-osd</li>
-<li>Document VNF impact while ceph is unavailable and once ceph service is restoredbeing restored.</li>
-</ul>
-</li>
-</ul>
-</li>
-<ul>
-<li><span style="text-decoration: underline;"><strong>Security - see Ch 7 for complete list</strong></span>
-<ul>
-<li>Validation above is performed using both RBAC Roles and User group policies, both of Admin and User Roles.
-</ul>
-<li>**SUMMARY:** Validate User Role to ensure it allow user to perform all designated task and prohibits user performing any unassigned task.</li>
-<li>Validate Security Policy/Rules are enforced</li>
-</ul>
-</li>
-</ul>
-</ul>
+**FuncTest**
+
+FuncTest test suites must be run as part of NFVi validation against all OpenStack based reference implementations. At a minimum, FuncTest executions will include execution of the functest-smoke and functional-benchmarking test suites.
+
+Additional Resources for FuncTest:
+- [Project Description](https://wiki.opnfv.org/display/functest/Opnfv+Functional+Testing#OpnfvFunctionalTesting-Testcases)
+- [User Guide](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/index.html)
+- [Test Suite Overview](https://opnfv-functest.readthedocs.io/en/stable-hunter/testing/user/userguide/test_overview.html)
+
+Example, or Reference, Functest Status Reporting Artifacts:
+- [Rally Verification](http://artifacts.opnfv.org/functest/functest-opnfv-functest-smoke-iruya-tempest_full-run-259/results/tempest_full/tempest-report.html)
+- [Rally Tasks](http://artifacts.opnfv.org/functest/functest-opnfv-functest-benchmarking-iruya-rally_full-run-169/results/rally_full/rally_full.html)
+
+**Baremetal**
+**SUMMARY:** Control and compute node hardware, bios, firmware, network interfaces, and base operating system configuration must be validated.
+- Interface - Validate nic status for all member in bond1 group
+- Interface - MTU speed for bond1 interface
+- Grub - SR-IOV is enabled
+- Numa - Validate numa configuration on each node
+- Numa - Validate total memory available is distributed equally across numa boundaries
+- VFs - Validate VFs are availablebeing created on each PCI SYS Interface.
+- Modules - Validate the following modules are loaded
+  - bonding
+  - 8021q
+  - i40e
+- OS: IPTables are enabled.
+- OS: ntp is enabled and configured to sync system time
+- OS: Dedicate mount point for following file system
+  - /var/log
+  - /var/lib/nova
+- OS: Validate Huge Pages are configured and enabled based on target infrastructure. (not available for all flavours)
+  - Validate Hugepage size configuration
+  - Validate Hugepage total number configuration
+- OS: Validate proxy/iptables implementation
+- OS: Validate proxy/iptables implementation
+- Validate proxy configuration after server reboot
+- Validate routing table after server reboot
+- BIOS: Validate Boot order
+- Validate Kernel version</li>
+- Validate BIOS version
+- Validate Firmware version
```

Comment:
This is too HW specific.

Reply:
Again, what HW doesn't have firmware, BIOS, or kernel? How are these test cases HW specific?

Reply:
If the testcase checks if the BIOS or firmware versions can be read that it okay. (Maybe the title should be Validate Firmware/BIOS version reading) If the test case is about checking if a BIOS/Firmware version equals v6.00XY that is not okay.

Reply:
See previous update, adding "against reference implementation"